docker/build.bash

```diff
@@ -37,14 +37,10 @@ fi
 user_id=$(id -u)
 image_name=$(basename $1)
 image_plus_tag=$image_name:$(date +%Y_%b_%d_%H%M)
-# mercurial adds a + symbol if there are uncomitted changes in the repo
-# that will break docker tag syntax
-hg_id=$(hg id -i | tr -d '+')
```

Comment:
When I ported this, I ended up using:

`git_id=$(git describe)` 

It requires you to create annotated tags with `git tag -a` and make sure they are pushed with `git push --tags`

Reply:
And it generates version numbers like `0.0.1-14-g136b05b` which is `$(last tag)-$(commits since tag)-g$(commit hash)`

Reply:
Thanks. I removed it because we don't use these tagged images, so it felt like a useless chunk of code.